revampedClient/components/cards/Card.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import '../../assets/components/_card.scss';
+
+const Card = (props) => {
+  const {
+    children, height, width,
+  } = props;
+
+  return (
+    <div
+      className="card"
+      style={{
```

Comment:
I think it will be a good approach to pass style as props in className in steady of allowing height and width to be passed in as props. 
Something like this `<div className={`card ${classes}`>`

Reply:
Great work @oluwajuwon, implement @Luckzman's feedback as this will make the component much more dynamic and reusable

Reply:
hmmm, that makes sense @Luckzman I'll look into that